.expeditor/config.yml

```diff
@@ -15,9 +15,8 @@ rubygems:
 docker_images:
   - chefdk
 
-habitat_packages:
-  - chef-dk:
-      also_build_for_linux_kernel2: true
```

Comment:
By default, unless you've specified a separate kernel2 plan file, we'll only build `x86_64-linux`. You can get around this by adding a `.bldr.toml` and specifying the targets you'd like to build on like https://github.com/chef/chef/blob/master/.bldr.toml

Reply:
Added